config/rbac/vg_manager_role.yaml

```diff
@@ -41,3 +41,11 @@ rules:
   - securitycontextconstraints
   verbs:
   - use
+- apiGroups:
```

Comment:
Was this generated by kubebuilder? If not please add that to the code.

Reply:
- no, it's custom added.
- anything generated by kubebuilder is used by operator and these are for vgmanager